feature-utils/poly-look/package.json

```diff
@@ -13,7 +13,7 @@
     "test:watch": "web-test-runner --watch",
     "build": "rollup -c rollup.config.mjs",
     "build-downstream": "../../build.js --start @polypoly-eu/poly-look",
-    "doc": "jsdoc -R README.md -r src -d ./docs"
+    "doc": "jsdoc -c jsdoc.conf.json"
```

Comment:
If this is the only change, how come package-lock.json have changed?

Reply:
I had initially installed the better-docs locally but removed it - seems the .lock was affected but didnt change after the remove package. will revert

Reply:
But it's used anyway, right? The templates are used?

Reply:
if you mean this change in package.json, yes it's used. the .lock file is reverted indeed

Reply:
No, I meant the better-docs package.